### ISSUE TITLE
[Enhancement] Present carts

### ISSUE THREAD

**Issue by hron84** [BrassGoggledCoders/MoarCarts#22]
Pleae add Chisel 1/2/3 present chest carts. They should be classic chest carts but different skin. I think @tterrag1098 will help you with the texture if you ask him.

**Comment by SkySom** [comment]
Could do the Present Chest carts. Can talk a look at Chisel.

**Comment by hron84** [comment]
[Container](https://github.com/Chisel-Team/Chisel/blob/1.7/dev/src/main/java/team/chisel/inventory/ContainerPresent.java) and [block](https://github.com/Chisel-Team/Chisel/blob/1.7/dev/src/main/java/team/chisel/block/BlockPresent.java)

**Comment by SkySom** [comment]
It doesn't appear though that 1.8+ Chisel has the Presents?

**Comment by hron84** [comment]
IIRC both of them has (1.7 and 1.8). But 1.7 definitely should be.

**Comment by hron84** [comment]
@SkySom if you looking for the registration code, it happens about [here](https://github.com/Chisel-Team/Chisel/blob/1.7/dev/src/main/java/team/chisel/Features.java#L1977). Present textures are decided based on metadata, similar how IronChest code works should be enough.

**Comment by SkySom** [comment]
https://github.com/Chisel-Team/Chisel/blob/1.8/dev/src/main/java/team/chisel/Features.java#L805 Looks like maybe 1.8 has it. (Note you keep pointing to 1.7. 1.7 is no longer having features added, it is bug fixes only)

**Comment by hron84** [comment]
> 1.7 is no longer having features added, it is bug fixes only)

So you stopped supporting MoarCarts on 1.7 at least with new features?

**Comment by SkySom** [comment]
Correct. Bug fixes only, no features. 1.8.9 is the current version receiving any potential features, with 1.9 being that version before long.

**Comment by hron84** [comment]
:disappointed: but thanks for the info. I hope Railcraft will come out to 1.8.9/1.9 soon.

**Comment by SkySom** [comment]
It's in the process.

**Comment by SkySom** [comment]
There is also the fact that Vanilla Hoppers do work with Minecarts, so loaders are still a thing even without Railcraft.
